common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
@doctoryes I created a branch, reverted @nedbat's commit and then created this PR against it. So all the migration is visible in this PR. I thought it would give us a picture of all of the transaction related logic in one place.